src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md

```diff
@@ -20,6 +20,7 @@ skill: beginner
 lang: en
 sidebar: true
 published: 2021-10-25
+postMergeBannerTranslation: page-upgrades-post-merge-banner-tutorial-ood
```

Comment:
Should a banner be displaying for these tutorial pages? I'm not seeing anything show up, e.g.
https://build-eae1f9b7-2544-4fb3-9d3f-abe2b5e5c4e7.gtsb.io/en/developers/tutorials/hello-world-smart-contract-fullstack/

Reply:
Fixed this and pushed into this branch. Should be good to test now